doc/faq/howto_faq.rst

```diff
@@ -442,10 +441,10 @@ Use :func:`~matplotlib.pyplot.show`
 -----------------------------------
 
 When you want to view your plots on your display,
-the user interface backend will need to start the GUI mainloop.
+the user interface backend will need to start the GUI main loop.
```

Comment:
Will someone who is a GUI expert weigh in on the correct nomenclature?

Reply:
I think splitting is correct (https://en.wikipedia.org/wiki/Event_loop).